data/SampleData/Hl7v2/ADT01-28.hl7

```diff
@@ -1,4 +1,4 @@
-﻿MSH|^~\&|ADT1|GOOD HEALTH HOSPITAL|GHH LAB, INC.|GOOD HEALTH HOSPITAL|198810181126|SECURITY|ADT^A01^ADT_A01|MSG00001|P|2.5.1||
+﻿MSH|^~\&|ADT1|GOOD HEALTH HOSPITAL|GHH LAB, INC.|GOOD HEALTH HOSPITAL|198808181126|SECURITY|ADT^A01^ADT_A01|MSG00001|P|2.5.1||
```

Comment:
What's the special meaning of changing the month from 10 to 08 here?

Reply:
We have discussed it offline. The data changed before and we change it back here.